lib/Basic/ConstructSolverChain.cpp

```diff
@@ -10,11 +10,12 @@
 /*
  * This file groups declarations that are common to both KLEE and Kleaver.
  */
-#include "klee/Common.h"
-#include "klee/CommandLine.h"
-#include "klee/Internal/Support/ErrorHandling.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include "../../include/klee/Basic/CommandLine.h"
+#include "../../include/klee/Basic/Common.h"
```

Comment:
Oh no! More relative includes! These should be `#include "klee/Basic/CommandLine.h` and `#include "klee/Basic/Common.h"`.